Sources/OpenTelemetryApi/Baggage/BaggageBuilder.swift

```diff
@@ -43,6 +43,6 @@ public protocol BaggageBuilder: AnyObject {
     /// - Parameter key: the EntryKey which will be removed.
     @discardableResult func remove(key: EntryKey) -> Self
 
-    /// Creates a Baggage from this builder.
-    func build() -> Baggage
+    /// Creates a Baggage from this builder or nil if it has no entries itself or the parent
+    func build() -> Baggage?
```

Comment:
returning nil may increase possibility for errors in using the library. We do it in .NET anyway. In Java we always return empty objects instead to minimize NRE. Raising comment to review if this is indeed what is wanted.

Reply:
I was returning nil to avoid forcing to return an EmptyBaggage class,  which I thought I had to export. But after thinking more about it and checking that I can still the empty class internal, I am reverting this change.